### PR TITLE
feat : added product page

### PR DESCRIPTION
**Pull request by sach10-create**

### CODE REVIEW

File: store/pages/product-page.html [hunk 1]
```diff
@@ -0,0 +1,409 @@
...
+<head>
+    <title>Nova Barca Store</title>
+    <meta charset="UTF-8" />
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.0.0-beta3/css/all.min.css"
+        integrity="sha512-Fo3rlrZj/k7ujTnHg4CGR2D7kSs0v4LLanw2qksYuRlEzO+tcaEPQogQ0KaoGN26/zrn20ImR1DfuLWnOo7aBA=="
+        crossorigin="anonymous" referrerpolicy="no-referrer" />
+    <link rel="stylesheet" href="https://ui-nova.netlify.app/css/component.css" />
+    <link rel="stylesheet" href="../../css/product-page.css">
+    <link rel="stylesheet" href="../../css/home.css">
+</head>
```
Comment (abhishektanwar) [inline store/pages/product-page.html:13]: suggestion : maybe use product-page.css after home.css just incase if some css from home.css has to be overridden

File: store/pages/product-page.html [hunk 2]
```diff
@@ -0,0 +1,409 @@
...
+                        class="no-list">
+                    <li class="no-list"><input type="checkbox" value="₹5000+" /> ₹5000+</li class="no-list">
+                </ul>
+
+                <ul class="checkbox-btn-container">
+                    <li class="no-list form-heading">Rating</li class="no-list">
+                    <li class="no-list"><input type="checkbox" value="4 and above" /> 4 and above</li class="no-list">
+                    <li class="no-list"><input type="checkbox" value="3 and above" /> 3 and above</li class="no-list">
+                    <li class="no-list"><input type="checkbox" value="2 and above" /> 2 and above</li class="no-list">
+                    <li class="no-list"><input type="checkbox" value="1 and above" /> 1 and above</li class="no-list">
```
Comment (abhishektanwar) [inline store/pages/product-page.html:122]: any reason to add class to li closing tag?

**Review by abhishektanwar — APPROVED** [review]
Good to merge.

File: store/pages/product-page.html (hunk 1 above)
Comment (sach10-create) [reply store/pages/product-page.html:13]: Thanks

File: store/pages/product-page.html (hunk 2 above)
Comment (sach10-create) [reply store/pages/product-page.html:122]: Typo

**Review by Omkar-Ghate — APPROVED** [review]
Good to merge.

**Pull request merged**